src/wp-includes/html-api/class-wp-html-tag-processor.php

```diff
@@ -1039,13 +1040,40 @@ private function parse_next_tag() {
 					'-' === $html[ $at + 2 ] &&
 					'-' === $html[ $at + 3 ]
 				) {
-					$closer_at = strpos( $html, '-->', $at + 4 );
-					if ( false === $closer_at ) {
-						return false;
+					$closer_at = $at + 4;
+
+					/*
+					 * Abruptly-closed empty comments are a sequence of dashes followed by `>`.
+					 */
```

Comment:
```suggestion
					 // Abruptly-closed empty comments are a sequence of dashes followed by `>`.
```
This should use the single line comment format.

Reply:
inlined in ddcb554583